invoke/exceptions.py

```diff
@@ -111,6 +111,18 @@ def __repr__(self):
         )
 
 
+class CommandTimedOut(UnexpectedExit):
+    def __init__(self, result, reason=None, timeout=None):
```

Comment:
`reason` is unused

Reply:
I mean when raising this exception you don't pass reason and the default is always used. If so, we don't need it in constructor arguments

Reply:
removed